data-requirements/summary-compare.ts

```diff
@@ -179,13 +197,14 @@ async function main() {
       elmParserDRByKey.forEach(dr => {
         const elmParserMustSupports = dr.mustSupport;
```

Comment:
```suggestion
        const elmParserMustSupports = dr.mustSupport?.filter(ms => !ms.includes('.'));
```
I'm currently filtering out secondary properties when doing the comparison. I think we could technically go either way on this one, but thought I'd note it here for consideration.

Reply:
Since the goal of the data requirements comparison script is to see the mustSupports output of both `fqm-execution` and `elm-parser-for-ecqms`, I am a little hesitant to do this filtering because then we are not seeing the full output for the `elm-parser-for-ecqms`? I think I am potentially missing some context here though, so I am happy to discuss more!